spec/features/listing_a_property_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature "listing a space" do
+  scenario "listing a space on /space/new route" do
+    visit("/spaces/new")
+    #save_and_open_page
+    fill_in("property_name", with: "A beautiful house")
+    fill_in("description", with: "A beautiful house by the river")
+    click_button "List my Space"
+
+    expect(current_path).to eq "/spaces"
+    expect(page).to have_content "A beautiful house"
+    expect(page).to have_content "A beautiful house by the river"
```

Comment:
Very similar descriptions. It might be finding the first "A beautiful house" in "A beautiful house by the river" instead of as a separate listing.

Reply:
we add another word in between